backend/api/src/http.rs

```diff
@@ -238,6 +241,11 @@ fn default_route() -> HttpResponse {
     ))
 }
 
+fn no_content_response() -> HttpResponse {
+    actix_web::HttpResponse::build(http::StatusCode::NO_CONTENT)
+        .finish()
+}
+
```

Comment:
Routes should avoid returning `HttpResponse` (paperclip can't tell what it is), instead, routes that return nothing should use [`paperclip::actix::NoContent`](https://paperclip.waffles.space/paperclip/actix/struct.NoContent.html)
```suggestion
fn no_content_response() -> NoContent {
    NoContent
}
```
Additionally, for consistency, prefer names like `get_base` or `index` over `no_content_response`.